examples/conservative-raster/triangle_and_lines.wgsl

```diff
@@ -0,0 +1,17 @@
+[[stage(vertex)]]
+fn vs_main([[builtin(vertex_index)]] vertex_index: u32) -> [[builtin(position)]] vec4<f32> {
+    var i: i32 = i32(vertex_index % 3u);
```

Comment:
let's use `const` here in other places instead of `var`

Reply:
ah. Haven't done much of this fancy new wglsl :) fixing!